Removing allinone class

avitacco opened the pull request:
As discussed in issue #76, I am removing the allinone class and the associated unit test.

## Notes for Reviewers

- [ ] The commit history must be preserved - please use the rebase-merge or standard merge option instead of squash-merge
- [ ] Sync up with the author before merging

bernd reviewed (commented):
Thank you! Can you also remove the test code in the `tests/` folder and the `Vagrantfile`file? I was using the code for manual testing with Vagrant. Since we now have the litmus tests, we don't need that anymore.

bernd commented:
Oh, and please remove the allinone documentation from the README.md file. Thanks!

avitacco commented:
Absolutely! I'm sorry I didn't catch these things initially, especially updating the documentation.

avitacco commented:
I removed the Vagrant-based tests and the allinone documentation from the readme. I also added some info in the readme about how to test using pdk and litmus. I also added details for using pdk in the contributing file. I hope this is acceptable.

bernd reviewed (approved):
Awesome, thank you!
